Automattic/jetpack
# AI Client: Add keyboard shortcuts in `AI Control`

Include shortcuts to `Send` and `Accept` in `AI Control` component.

Now you do the actions with the following keys:

`Newline` - `Shift` + `Enter`
`Send` - `Enter`
`Accept` - `CMD/CTRL + Enter` (only if Accept button is visible)

## Proposed changes:

* Add `@wordpress/compose` and `@storybook/addon-actions` packages.
* Add `useKeyboardShortcuts` hook to map the keys.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


N/A

## Testing instructions:






* Start `Storybook`
* Goes to `AI Client/AI Control` story.
* First check if `newline` works using: `Shift + Enter`.
* Now opens `Actions` tab.
* Check if `send` works using: `Enter`, it will show `onSend` in Actions.
* Check if nothing happens when using `CMD/CTRL + Enter`.
* Goes to `Control` tab and enable `showAccept`.
* Back to `Actions`
* Check if `accept` works using: `CMD/CTRL + Enter`, it will show `onAccept` in Actions.
